tests/test_utils.py

```diff
@@ -64,11 +64,13 @@ def test_get_alias_dict():
 
     assert alias_regression["c(rt, response)"] == "rt,response"
     assert alias_regression["Intercept"] == "v_Intercept"
+    assert alias_regression["x"] == "v_x"
```

Comment:
is this test_util still ok after demoting `v` from default parent?

Reply:
Yep. In this case the parent is still `"v"`. With `alias_regression_a` the parent becomes `"a"` which is why I had to change the tests